package.json

```
@@ -110,7 +110,7 @@
     "rxjs": "^7.3.0",
     "swagger-ui-express": "^4.1.4",
     "typechain": "^7.0.1",
-    "typeorm": "^0.2.41",
+    "typeorm": "^0.3.0",
```

```suggestion
    "typeorm": "0.2.41",
```

---

Let it stay latest compatible with 0.2
```suggestion
    "typeorm": "^0.2.41",
```